### PR TITLE
User API Fixing role param

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Fixes `role` param for user creation.

### Motivation
Feedback from dev team

### Preview link

* https://docs-staging.datadoghq.com/gus/api-user/api/?lang=python#create-user
* https://docs-staging.datadoghq.com/gus/api-user/api/?lang=python#update-user

### CODE REVIEW

File: content/api/users/code_snippets/api-user-create.py [hunk 1]
```diff
@@ -8,4 +8,4 @@
 initialize(**options)
 
 # Create user
-api.User.create(handle='test@datadoghq.com', name='test user')
+api.User.create(handle='test@datadoghq.com', name='test user', role='st')
```
Comment (MLaureB) [inline content/api/users/code_snippets/api-user-create.py]: The param for the role is named `access_role` (not `role`).

File: content/api/users/code_snippets/api-user-update.rb [hunk 2]
```diff
@@ -6,4 +6,4 @@
 
 dog = Dogapi::Client.new(api_key, app_key)
 
-dog.update_user('test@datadoghq.com', : email => 'test+1@datadoghq.com', : name => 'alt name')
\ No newline at end of file
+dog.update_user('test@datadoghq.com', : email => 'test+1@datadoghq.com', : name => 'alt name', : acces_role => 'ro')
```
Comment (cswatt) [inline content/api/users/code_snippets/api-user-update.rb]: r/acces_role/access_role

**Comment by l0k0ms** [comment]
@cswatt the `access_role` parameter is correct here: 

```
curl -X POST -H "Content-type: application/json" \                                                                                                                                                                  
   -d '{"handle":"afezef@mailinator.com","name":"test user", "access_role":"adm"}' \
   "https://api.datadoghq.com/api/v1/user?api_key=XXXXX&application_key=XXX"

{"user":{"disabled":false,"handle":"afezezf@mailinator.com","name":"test user","is_admin":true,"role":null,"access_role":"adm","verified":false,"email":"afezef@mailinator.com","icon":"https://secure.gravatar.com/avatar/aa8371b67930a7959bf878144cf16361?s=48&d=retro"}}%                                                                                                                                                                                                              
```

**Comment by cswatt** [comment]
it's mis-spelled in `api-user-update.rb` though

File: content/api/users/code_snippets/api-user-update.rb (hunk 2 above)
Comment (l0k0ms) [reply content/api/users/code_snippets/api-user-update.rb]: well spotted

**Pull request merged**